### PR TITLE
Fix bug in web runner

### PR DESCRIPTION
**Pull request by MeirDimant**

### CODE REVIEW

File: Infrastructure/ContainerBuilder.cs [hunk 1]
```diff
@@ -9,12 +9,13 @@
 using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
-using List;
+using List;  
 using CountDown;
```
Comment (YacovGold) [inline Infrastructure/ContainerBuilder.cs]: Please format the code...

File: Runners/WebRunner/Controllers/WhatsappBotController.cs [hunk 2]
```diff
@@ -20,7 +20,7 @@ public class WhatsappBotController : TwilioController
 
         public WhatsappBotController(PluginExecutor pluginExecutor)
         {
-            _pluginExecutor ??= pluginExecutor;
+            _pluginExecutor = pluginExecutor;
         }
```
Comment (YacovGold) [inline Runners/WebRunner/Controllers/WhatsappBotController.cs:24]: Maybe also do the same to the other controllers

**Comment by YacovGold** [comment]
What fix exactly?

File: Runners/WebRunner/Controllers/WhatsappBotController.cs (hunk 2 above)
Comment (MeirDimant) [reply Runners/WebRunner/Controllers/WhatsappBotController.cs:24]: It is the same in other controllers

**Pull request merged**